ClientApp/viking-vault/src/AttachCardModal.css

```diff
@@ -0,0 +1,61 @@
+.modal-content.attach-card {
+    border-radius: 0.95rem;
+}
+
+.form-group.attach-card {
+    width: 86%;
+    margin-left: 7%;
```

Comment:
Are you trying to center the element by using this margin?

Reply:
Yes, that's what I wanted to do. I'll modify the seventh line, replacing it with margin: auto.

Reply:
Yes, please use `margin: auto` when you want to center something instead of using percentages margins. Alternatively, you can also use flexbox to center both vertically and horizontally